android/app/build.gradle

```diff
@@ -123,13 +123,13 @@ android {
             include "armeabi-v7a", "x86"
         }
     }
-    buildTypes {
-        release {
-            minifyEnabled enableProguardInReleaseBuilds
-            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
-            signingConfig signingConfigs.release
-        }
-    }
+    // buildTypes {
+    //     release {
+    //         minifyEnabled enableProguardInReleaseBuilds
+    //         proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+    //         signingConfig signingConfigs.release
```

Comment:
Hm after reading this now, the permenant solution would probalby be to add a buildType called dev that that doesn't rely on signing.